docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -0,0 +1,136 @@
+# Blockchain API Server
+
+## Blockchain RPC
+
+Blockchain RPC API for interacting with different blockchains can be found in a separate `Blockchain RPC` subsection.
+
+## Hexless account profiles API
+
+Email wallet users can register a profile name e.g. `coolname.connect.id` and use it instead of the `0x` address.
+Server API exposes forward lookup for the `name.connect.id` -> `0x` address with chain IDs and reverse lookup for the `0x` -> `name.connect.id`.
+Users can also update the name data including avatar, description text and bind addresses.
+
+### Name lookup
+
+Used to lookup address for the name.
+
+`GET /v1/profile/account/{name}`
+
+* `name` - is the registered name. eg. `coolname.connect.id`
+
+#### Success response body:
+
+```jsonc
+{
+    "name": string,
+    "address": string,
+    "text": (optional)string,
+    "avatar": (optional)string,
+    "registered": datetime,
+    "updated": datetime,
+    "chainIDs" integer[]
```

Comment:
Does optional mean the field might not exist or it's nullable? Let's make it explicit. Also there is no datetime or integer in JSON, just numbers or strings, so again let's specify that.

Also `chain_ids` as a stylistic preference.

```suggestion
```typescript
{
    "name": string,
    "address": string,
    "text": string | null,
    "avatar": string | null,
    "registered": number,
    "updated": number,
    "chain_ids" number[]
```

Reply:
Makes sense to change it to this format. Done.